### PR TITLE
Dev/refactor

### PR DESCRIPTION
**Pull request by cfrontin**

### CODE REVIEW

File: hopp/to_organize/run_profast_for_ammonia.py [hunk 1]
```diff
@@ -106,14 +106,14 @@ def run_profast_for_ammonia(plant_capacity_kgpy,plant_capacity_factor,plant_life
     pf.set_params('capital gains tax rate',0.15)
     pf.set_params('sell undepreciated cap',True)
     pf.set_params('tax losses monetized',True)
-   # pf.set_params('operating incentives taxable',True)
+    pf.set_params('annual operating incentive', {"value":163673.7733, "decay":0.034906267, "sunset years":6, "taxable":True})
```
Comment (jaredthomas68) [inline hopp/to_organize/run_profast_for_ammonia.py:109]: Do we need this? Is this specific to certain tech? Why is the value hard coded?

File: hopp/to_organize/run_profast_for_h2_transmission.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ def run_profast_for_h2_transmission(project_dir, max_hydrogen_production_rate_kg
     pf.set_params('capital gains tax rate',0.15)
     pf.set_params('sell undepreciated cap',True)
     pf.set_params('tax losses monetized',True)
-    #pf.set_params('operating incentives taxable',True)
+    pf.set_params('annual operating incentive', {"value":163673.7733, "decay":0.034906267, "sunset years":6, "taxable":True})
```
Comment (jaredthomas68) [inline hopp/to_organize/run_profast_for_h2_transmission.py:84]: same comments as before (see above at line 109 in hopp/eco/hopp_mgmt.py)

File: hopp/to_organize/run_profast_for_hydrogen.py [hunk 3]
```diff
@@ -107,11 +107,8 @@ def run_profast_for_hydrogen(hopp_dict,electrolyzer_size_mw,H2_Results,\
     property_tax_insurance = 1.5/100    #[% of Cap/y]
     variable_OM = 1.30  #[$/MWh]
 
-    H2_PTC_duration = 10 # years the tax credit is active
-    Ren_PTC_duration = 10 # years the tax credit is active
-
     electrolysis_total_EI_policy_grid,electrolysis_total_EI_policy_offgrid\
-          = LCA_single_scenario_ProFAST.hydrogen_LCA_singlescenario_ProFAST(grid_connection_scenario,atb_year,site_name,policy_option,hydrogen_production_while_running,H2_Results,electrolyzer_energy_kWh_per_kg,solar_size_mw,storage_size_mw,hopp_dict,H2_PTC_duration)
+          = LCA_single_scenario_ProFAST.hydrogen_LCA_singlescenario_ProFAST(grid_connection_scenario,atb_year,site_name,policy_option,hydrogen_production_while_running,H2_Results,electrolyzer_energy_kWh_per_kg,solar_size_mw,storage_size_mw,hopp_dict)
```
Comment (jaredthomas68) [inline hopp/to_organize/run_profast_for_hydrogen.py:111]: why is the duration being removed?

File: hopp/to_organize/run_profast_for_hydrogen.py [hunk 4]
```diff
@@ -120,35 +117,50 @@ def run_profast_for_hydrogen(hopp_dict,electrolyzer_size_mw,H2_Results,\
```
Comment (jaredthomas68) [inline hopp/to_organize/run_profast_for_hydrogen.py:117]: Where are these changes coming from?

File: hopp/to_organize/run_profast_for_hydrogen.py [hunk 5]
```diff
@@ -216,136 +228,119 @@ def run_profast_for_hydrogen(hopp_dict,electrolyzer_size_mw,H2_Results,\
...
-    install_years = 3
-    analysis_start = atb_year + 5 - install_years
-
     pf.set_params('commodity',{"name":'Hydrogen',"unit":"kg","initial price":100,"escalation":gen_inflation})
     pf.set_params('capacity',electrolysis_plant_capacity_kgperday) #units/day
     pf.set_params('maintenance',{"value":0,"escalation":gen_inflation})
     pf.set_params('analysis start year',2022)
     pf.set_params('operating life',plant_life)
-    pf.set_params('installation months',install_years*12)
+    pf.set_params('installation months',36)
```
Comment (jaredthomas68) [inline hopp/to_organize/run_profast_for_hydrogen.py:343]: why are we going from variable to hard-coded?

File: hopp/to_organize/run_profast_for_steel.py [hunk 6]
```diff
@@ -163,14 +163,14 @@ def run_profast_for_steel(plant_capacity_mtpy,plant_capacity_factor,\
     pf.set_params('capital gains tax rate',0.15)
     pf.set_params('sell undepreciated cap',True)
     pf.set_params('tax losses monetized',True)
-    #pf.set_params('operating incentives taxable',True)
+    pf.set_params('annual operating incentive', {"value":163673.7733, "decay":0.034906267, "sunset years":6, "taxable":True})
```
Comment (jaredthomas68) [inline hopp/to_organize/run_profast_for_steel.py:166]: again, is this tech specific and why is it hard-coded?

**Review by jaredthomas68 — COMMENTED** [review]
A few questions that need to be answered

**Comment by cfrontin** [comment]
I've worked through these, and have eliminated the spurious deletions.

I now have some issues running the phase1 code like:
```
(hopp) cfrontin-38617s:phase1 cfrontin$ python green_steel_ammonia_define_scenarios.py 

...

Traceback (most recent call last):
  File "/Users/cfrontin/projects/green_steel/GreenSteel/src/phase1/green_steel_ammonia_define_scenarios.py", line 131, in <module>
    batch_generator_kernel(arg_list[runs])
  File "/Users/cfrontin/projects/green_steel/GreenSteel/src/phase1/green_steel_ammonia_run_scenarios.py", line 911, in batch_generator_kernel
    h2_solution,h2_summary,profast_h2_price_breakdown,lcoh_breakdown,electrolyzer_installed_cost_kw,elec_cf,ren_frac,electrolysis_total_EI_policy_grid,electrolysis_total_EI_policy_offgrid,H2_PTC,Ren_PTC,h2_production_capex = run_profast_for_hydrogen. run_profast_for_hydrogen(hopp_dict,electrolyzer_size_mw,H2_Results,\
TypeError: run_profast_for_hydrogen() missing 1 required positional argument: 'use_optimistic_pem_efficiency'
```

**Pull request closed without merging**